ribxlib/tests/test_parsers.py

```diff
@@ -44,6 +50,37 @@ def test_work_impossible(self):
             'Explanation in attribute' in instance.work_impossible)
 
 
+class TestInspectionPipeParserRibx_13(unittest.TestCase):
+    """Test regarding version 1.3 of Ribx."""
+
+    def test_ribx_13_smoke(self):
+        """This ribx has two inspection headers with the same Pipe ref. For
+        inspection of Pipes the manhole_start attribute must be set.
+        """
+        mode = Mode.INSPECTION
+        f = os.path.join(RIBX13_DATA_DIR, "36190148 5300093.ribx")
+        ribx, log = parse(f, mode)
+        # self.assertFalse(log)
+        self.assertFalse(len(log))
+
+        p0, p1 = ribx.inspection_pipes  # there are only 2 pipes (see file)
+
+        self.assertEqual(p0.ref, p1.ref)
```

Comment:
Meerdere tests in één testfunctie hoort eigenlijk niet. Als het echt twee dezelfde soort checks zijn en het is irritant de testcode te kopiëren, dan doe ik het zelf ook wel eens. Maar in deze test staan er drie. Da's een beetje veel.